### PR TITLE
Core: Cleanup

### PR DESCRIPTION
**Pull request by jronak**

### CODE REVIEW

File: core/package.json [hunk 1]
```diff
@@ -28,6 +24,6 @@
     "url": "git@github.com:juspay/presto-ui.git"
   },
   "author": "JUSPAY Technologies <info@juspay.in>",
-  "license": "AGPL",
+  "license": "AGPL-3.0",
```
Comment (dilipzotha) [inline core/package.json:27]: This is AGPL.

File: core/src/init.js [hunk 2]
```diff
@@ -218,22 +215,14 @@ const runDuiCallback = (state) => {
     callback(state)();
 };
 
-let purescriptInit = (meta, purescriptObj_) => {
+let purescriptInit = (meta, main) => {
```
Comment (dilipzotha) [inline core/src/init.js]: Let us move this out in next release.

File: core/package.json (hunk 1 above)
Comment (jronak) [reply core/package.json:27]: There is no valid license name as AGPL.
We use AGPL-3.0.
https://github.com/juspay/presto-ui/blob/master/LICENSE#L24

Identifier: https://spdx.org/licenses/AGPL-3.0.html

File: core/src/init.js (hunk 2 above)
Comment (jronak) [reply core/src/init.js]: ui-generator expects this version.

**Pull request merged**